tools/interactive/interactivetool_geoexplorer.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="interactive_tool_geoexplorer" tool_type="interactive" name="geoexplorer" version="0.1">
+    <description>An interactive spatial analysis platform using ggvis and Leaflet</description>
+    <requirements>
+        <container type="docker">ylebras/geoexplorer-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="geoexplorer visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/sample-apps/SIG/</url>
+        </entry_point>
+    </entry_points>
+    <environment_variables>
+
+        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($outfile.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
```

Comment:
please use the new way to set those envs ... look at juypter or openrefine

Reply:
done